packages/dashboard-backend/src/api/serverConfigApi.ts

```diff
@@ -44,6 +48,7 @@ export function registerServerConfigApi(server: FastifyInstance) {
       pluginRegistry: {
         openVSXURL,
       },
+      cheNamespace: CHECLUSTER_CR_NAME,
```

Comment:
I'm a bit lost here, why CR name = namespace?

Reply:
Good catch. It is a typo. Thanks. It should be```CHECLUSTER_CR_NAMESPACE```
I have fixed it.